### PR TITLE
Add KBC_DATA_TYPE_SUPPORT env only if native types are enabled

### PR DESCRIPTION
**Pull request by ErikZigo**
Jira: https://keboola.atlassian.net/browse/PST-2467

Before asking for review, check the following:

- [ ] For any functionality change or the addition of a new feature, I verified if it should also be implemented in the no-DIND version. If so, I created a corresponding task [PST-XXX] under the [no-DIND epic](https://keboola.atlassian.net/browse/PST-918)

---

Viz ticket. Melo by vyresit edge case s tim, kdy na novych native types vytvoritm konfiguraci, vypnu si native types a pustim komponentu

### CODE REVIEW

File: Tests/backend-specific-tests/ABS/DataLoaderPersistentABSWorkspaceTest.php [hunk 1]
```diff
@@ -9,45 +9,18 @@
...
-    private Client $client;
-
-    public function setUp(): void
-    {
-        parent::setUp();
-
-        $this->client = new Client([
-            'url' => getenv('STORAGE_API_URL_SYNAPSE'),
-            'token' => getenv('STORAGE_API_TOKEN_SYNAPSE'),
-        ]);
```
Comment (ErikZigo) [inline Tests/backend-specific-tests/ABS/DataLoaderPersistentABSWorkspaceTest.php:41]: Synapse pro tenhle test není potřeba. Důležité je aby byl ABS file workspace.

Envy jsem fixoval tu https://github.com/keboola/docker-bundle/pull/779 takže tohle se může vyhodit a může se použit klient z `BaseDataLoaderTest`.

**Review by ondrajodas — APPROVED** [review]
jn je to jak jsme se domluvili 👍
